utilities/markdown.js

```diff
@@ -65,8 +65,8 @@ module.exports = function(section) {
     if(detailsEnabled) {
       rendered = rendered.replace(/MARKDOWNDETAILSSTART.*?\n/g, "<details>");
       rendered = rendered.replace(/\n.*?MARKDOWNDETAILSEND.*?\n/g, "</details>");
-      rendered = rendered.replace(/\n.*?MARKDOWNSUMMARYSTART.*?\n/g, "<summary>");
-      rendered = rendered.replace(/\n.*?MARKDOWNSUMMARYEND.*?\n/g, "</summary>");
+      rendered = rendered.replace(/\n.*?MARKDOWNSUMMARYSTART.*?\n/g, "<summary><span style='margin-left: -15px; cursor: pointer;'>");
```

Comment:
It might be nicer to tag the `span` with a class and handle this through CSS. Easier to style.